earth_enterprise/src/fusion/rasterfuse/gepackgen.cpp

```diff
@@ -85,6 +89,9 @@ int main(int argc, char *argv[]) {
   if (!imagery && !terrain) {
     usage(progname, "--imagery or --terrain is required");
   }
+  if (imagery && (decimation_threshold != 0.009)) {
```

Comment:
if (imagery && (decimation_threshold != defaultDecimationThreshold ))

Reply:
This one I agree with, I forgot to change this when i added defaultDecimationThreshold a couple hours ago